nuxeo-runtime/nuxeo-runtime-reload/src/main/java/org/nuxeo/runtime/reload/ReloadService.java

```diff
@@ -52,6 +52,15 @@
     String RELOAD_SEAM_EVENT_ID = "reloadSeamComponents";
 
     /**
+     * This property allows to use the former way to hot reload Nuxeo server.
+     * <p/>
+     * The property and the former mechanism will be removed in the next LTS version.
+     *
+     * @since 9.3
+     */
+    String USE_FORMER_HOT_RELOAD = "nuxeo.hotreload.former.mechanism";
```

Comment:
Using the "compat" word than "former" may be better (but it's a detail)

Reply:
Yeah good idea, I'll change that.